migrations/000003_create_books_table.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS books (
+    id SERIAL PRIMARY KEY,
+    title VARCHAR(255) NOT NULL,
+    isbn VARCHAR(13) UNIQUE NOT NULL,
+    author_id INT NOT NULL REFERENCES authors(id) ON DELETE CASCADE,
+    publisher VARCHAR(255),
+    price DECIMAL(10, 2) NOT NULL,
+    stock INT DEFAULT 0,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE INDEX IF NOT EXISTS idx_books_isbn ON books(isbn);
+CREATE INDEX IF NOT EXISTS idx_books_title ON books(title);
```

Comment:
It's better to use gin index with pg_vector, to fire full text search in future.